Mayhemfile

```diff
@@ -0,0 +1,5 @@
+project: trust-dns
+target: trust-dns
```

Comment:
Just a minor nitpick to ensure that the target name matches what is being fuzzed